packages/flutter/lib/src/gestures/events.dart

```diff
@@ -232,6 +233,8 @@ abstract class PointerEvent with Diagnosticable {
   }) : localPosition = localPosition ?? position,
        localDelta = localDelta ?? delta;
 
+  /// Corresponds to the motion event that resulted in this pointer event. (only android)
+  final int motionEventId;
```

Comment:
A name for this property that doesn't tie it to Android's MotionEvents (and hence can be reused for other platforms) would probably also be a good idea.